wasp/apps/alarm.py

```diff
@@ -96,12 +97,13 @@ def __init__(self):
             for alarm in alarms:
                 n = self.num_alarms
                 h, m, st = map(int, alarm.split(","))
-                self.alarms[n][0] = h
-                self.alarms[n][1] = m
-                self.alarms[n][2] = st
+                self.alarms[n][_HOUR_IDX] = h
+                self.alarms[n][_MIN_IDX] = m
+                self.alarms[n][_ENABLED_IDX] = st
                 self.num_alarms += 1
-        except Exception:
-            pass
+        except Exception as err:
+            wasp.system.switch(PagerApp("Error when reloading alarms: "
+                                         "'{}'".format(err)))
```

Comment:
Will this trigger also the first time the app is run, when the file is not there yet?

Reply:
Yes. My personnal inclination is of course to pay attention to this kind of things but on waspos given that every line seems to count I felt that was a good compromise.

What do you think?